client/packages_test.go

```diff
@@ -87,3 +89,45 @@ func (cs *clientSuite) TestClientPackages(c *check.C) {
 		},
 	})
 }
+
+const (
+	appID = "chatroom.ogra"
+)
+
+func (cs *clientSuite) TestClientPackageCallsEndpoint(c *check.C) {
+	_, _ = cs.cli.Package(appID)
+	c.Check(cs.req.Method, check.Equals, "GET")
+	c.Check(cs.req.URL.Path, check.Equals, fmt.Sprintf("/1.0/packages/%s", appID))
+}
+
+func (cs *clientSuite) TestClientPackage(c *check.C) {
+	cs.rsp = `{
+		"type": "sync",
+		"result": {
+			"description": "WebRTC Video chat server for Snappy",
+			"download_size": "6930947",
+			"icon": "/1.0/icons/chatroom.ogra/icon",
+			"installed_size": "18976651",
+			"name": "chatroom",
+			"origin": "ogra",
+			"resource": "/1.0/packages/chatroom.ogra",
+			"status": "active",
+			"type": "app",
+			"vendor": "",
+			"version": "0.1-8"
+		}
+	}`
+	pkg, err := cs.cli.Package(appID)
+	c.Check(err, check.IsNil)
```

Comment:
This should be an Assert.. if this fails, the rest of the test is not interesting and reduces the clarity of the error when debugging.